app/database/models/server/my_channel.ts

```diff
@@ -41,6 +41,9 @@ export default class MyChannelModel extends Model {
     /** is_unread : Whether the channel has unread messages */
     @field('is_unread') isUnread!: boolean;
 
+    /** is_muted : Whether the channel is muted */
+    @field('is_muted') isMuted!: boolean;
```

Comment:
Why do we need to add this to the database and the model? We already have the MyChannelSettings model that holds the channel notify_props, which in turn holds the `mark_unread` prop that is used to determine if the channel is muted or not.

basically you should check for `notifyProps.mark_unread === 'mention'` for the `MyChannelSettings` record.

Reply:
The answer is I don't know :P I'm still getting my head wrapped around everything and I was following the pattern of `is_unread` which seemed to be a similar case.

If that's the way to do it then I'll close the PR.

Reply:
fair enough.. yeah, for this particular case we should be using the data in `MyChannelSettings` modal to determine if the channel is muted or not, there is definitely no need to add a property to the model.

The current schema holds support for 100% of the existing features in Channels. it could be that something was missed though.